README.md

```diff
@@ -111,7 +111,8 @@ $timeout(function () {
 }
 ```
 
-While the promise is being resolved, a default text - "Working..." will show. In the configuration section you can learn how to change it.
+While the promise is being resolved, a default text - "Working..." will show. In the configuration section you can
```

Comment:
We've actually changed this, so that the default text is an empty string. This allows us to ensure the text change doesn't go from `old -> empty -> new`, but rather just `old -> new` by default. If the user does change the default text, then we'll show it in between the old and new.